npm/vue/cypress.config.js

```diff
@@ -0,0 +1,32 @@
+const { startDevServer } = require('@cypress/webpack-dev-server')
```

Comment:
Cool, in future we should try to use `git mv` to preserve the history. It will also make the PR smaller to review.

Reply:
In theory yes, but have you seen it actually work?

Reply:
Usually, but I had an experience recently where it did not - seems if you change too many LOC in the file, it doesn't work. Not really a big deal, but good to keep in mind.